src/zipObjWith.js

```diff
@@ -0,0 +1,31 @@
+import { curryN, pipe, apply, map, fromPairs, zip } from 'ramda';
+
+/**
+ * Creates a new object out of a list of keys and a list of values by applying the function
+ * to each equally-positioned pair in the lists.
+ * Key/value pairing is truncated to the length of the shorter of the two lists.
+ *
+ * @func zipObjWith
+ * @memberOf RA
+ * @category Object
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.22.0|v2.22.0}
+ * @sig (v, k) -> [k, v] -> [k] -> [v] -> { c: d }
```

Comment:
Signature dosn't seem right. It's not apparent where `c` and `d` comes from. So I'd opt for one of the following:
```js
@sig (v, k) -> [k, v] -> [k] -> [v] -> { k: v }
```
or
```js
@sig (b, a) -> [k, v] -> [a] -> [b] -> { k: v }
```

I think the second signature captures reality more. We have two list of arbitrary values, we apply the transformation function on those values which gives us `(k, v)` pairs, which are then transformed to `{ k: v }` object. It's implied by javascript implementation that `k is String`, but `a is *`. `k` is created by transforming `a` to String.

Reply:
Agreed- I think `c` and `d` are there from a previous attempt!